lookout/style/format/feature_extractor.py

```diff
@@ -259,14 +241,14 @@ def extract_features(self, files: Iterable[File], lines: List[List[int]]=None) \
                     traceback.print_exc()
                     input("Press Enter to continue…")
                 continue
-            file_vnodes = self._classify_vnodes(file_vnodes, file.path)
-            file_vnodes = self._merge_classes_to_composite_labels(
-                file_vnodes, file.path, index_labels=index_labels)
+            file_vnodes_iterator = self._classify_vnodes(file_vnodes, file.path)
+            file_vnodes_iterator = self._merge_classes_to_composite_labels(
+                file_vnodes_iterator, file.path, index_labels=index_labels)
             if self.insert_noops:
-                file_vnodes = self._add_noops(list(file_vnodes), file.path,
+                file_vnodes = self._add_noops(list(file_vnodes_iterator), file.path,
```

Comment:
`if self.insert_noops:` is still here... I recall that it is always True now, so what's the point?

Reply:
indeed, I removed it in the next PR but will back-port because it should be fixed ASAP